content/docs/0.6.0/reference/monitoring/dynatrace/index.md

```diff
@@ -187,7 +235,7 @@ The dynatrace-service in Keptn will take care of pushing events of the Keptn wor
     caption="Keptn events"
     width="500px">}}
 
-## (optional) Create process group naming rule in Dynatrace
+## Create process group naming rule in Dynatrace
```

Comment:
I think this should still be optional, or is it not?

Reply:
Setting up naming rules in DT is a best-practice. I would encourage Keptn-DT users to do this extra step for getting a better experience.

Reply:
I get your point, but that step affects all services, not just the one that are managed by Keptn, and it is not required for running Dynatrace Monitoring. In addition it is definately not required for the quality-gates (standalone) use-case (one could want to monitor a service that does not run within a Kubernetes cluster).

Anyway, we can remove the **(optional)** from the heading, but we should add a note below:

"While it is not a technical requirement, we recommend setting up a process group naming rule within Dynatrace". 

Also, while you are at it, can you remove the first enumeration (`1. Create a naming rule for process groups`) as it does not make sense to have that unelss we have a `2.` step somewhere below :)

Reply:
The hint is definitely worth to be mentioned. I added it and removed the 1.